frontend/src/features/map_planning/layers/base/components/BaseLayerRightToolbar.tsx

```diff
@@ -8,6 +9,16 @@ import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { FileStat } from 'webdav';
 
+function validateBaseLayerOptions(baseLayerOptions: Omit<BaseLayerImageDto, 'action_id'>) {
+  const { id, layer_id, path, rotation, scale } = baseLayerOptions;
+
+  if (!id || !layer_id || !path || !rotation || !scale) {
```

Comment:
Is this true if rotation is set to 0?

Reply:
The condition is evaluated as true yes. Bcs 0 is falsy in JS

Reply:
Already fixed in master via #776 and deployed.